javascript-if/main.js

```diff
@@ -0,0 +1,61 @@
+/* exported isUnderFive,
+            isEven,
+            startsWithJ,
+            isOldEnoughToDrink,
+            isOldEnoughToDrive,
+            isOldEnoughToDrinkAndDrive,
+            categorizeAcidity,
+            introduceWarnerBro
+ */
+function isUnderFive(num) {
+  return num < 5;
+}
+
+function isEven(num) {
+  return (num % 2 === 0);
+}
+
+function startsWithJ(string) {
+  return string[0] === 'J' || string[0] === 'j';
+}
```

Comment:
While it's good to think about multiple versions of the same letter, this particular function specifies a _capital_ `'J'`:

---

![image](https://user-images.githubusercontent.com/16143771/152250936-60565afd-1f85-4fdf-887a-035e511548b6.png)